node/config/def.go

```diff
@@ -324,6 +331,10 @@ func DefaultStorageMiner() *StorageMiner {
 			AlwaysKeepUnsealedCopy:    true,
 			FinalizeEarly:             false,
 
+			CollateralFromMinerBalance: false,
+			AvailableBalanceBuffer:     types.FIL(big.Zero()),
```

Comment:
Is 0 the best default here? I guess it's okay because `CollateralFromMinerBalance` is defaulting to false anyway.